scripts/release_testing_script/config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  gitHubApi: 'https://api.github.com/',
+  gitHub: 'https://github.com/',
+  projects: 'newtewt/release_testing_example/projects',
+  issuesEnd: 'repos/newtewt/release_testing_example/issues',
+  projectCreate: 'repos/newtewt/release_testing_example/projects',
+  repoName: 'release_testing_example',
+  owner: 'newtewt',
+
+  columnNames: ["To Do", "In Progress", "Done"],
+
+  headers: {
+
+    'User-Agent': 'newtewt',
+    'Accept': 'application/vnd.github.inertia-preview+json',
+    'Authorization': process.env.githubToken
```

Comment:
Other scripts use a token.json file (eg: `scripts/changelog-generator/index.js`). It probably makes sense to use the same thing so each engineer only needs to set it in one place.